src/components/Main/Main.tsx

```diff
@@ -125,7 +129,6 @@ function Main() {
 
       if (autorunSimulation) {
         updateBrowserURL(nextLocationSearch)
-        debouncedRun(allParams, scenarioState, severity)
       }
```

Comment:
Update browser URL parameters if location search changed and autorunSimulation is true.